### ISSUE TITLE
Release for Scala 2.12.x ?

### ISSUE THREAD

**Issue by ronanM** [rockymadden/stringmetric#31]
Do you plan to release a version for Scala `2.12.x` ?

**Comment by firehooper** [comment]
When you have a chance can you please do a release for https://github.com/rockymadden/stringmetric/pull/27 ?
Thanks for compiling all these great string matching algorithms!

**Comment by ndelaforge** [comment]
[This](https://github.com/Mnemotix/stringmetric) may help you.

**Comment by GRBurst** [comment]
@ndelaforge That fork is not available on maven using sbt. Using as suggested with
```
libraryDependencies += "com.rockymadden.stringmetric" %% "stringmetric-core" % "0.28.0-SNAPSHOT"
```
results in an error (could not resolve). It does not even work when adding sonatype as resolver.

Edit: Because of that I am using it with publish local, but I would prefer if it is available on maven :-)

**Comment by ndelaforge** [comment]
@GRBurst yes, I had an error with PGP publishSigned while pushing artifacts to sonatype.
I have no time to investigate further for the moment. I'll sort that ASAP.

Cheers.
